packages/forgetti/src/core/presets.ts

```diff
@@ -33,6 +33,7 @@ export interface Preset {
   };
   runtime: {
     useRef: ImportDefinition;
+    useMemo: ImportDefinition;
```

Comment:
Probably need to document this requirement in the README.

Reply:
`configuration.md` updated in [`0c23663` (#30)](https://github.com/lxsmnsyc/forgetti/pull/30/commits/0c2366308331ef4a8fa70536ac4c5034c8090970).